data_collection/web/javascript_tracking_client/api.rst

```diff
@@ -2309,3 +2309,57 @@ Miscellaneous
             .. code-block:: javascript
 
                 jstc.trackError(new Error("Uncaught SyntaxError"));
+
+.. function:: getTrackingSource()
+
+    Returns tracking source data identifying the library and passed in ``ts_n`` and ``ts_v`` tracking parameters.
+    The default tracking source can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```

Comment:
```suggestion
    Returns tracking source name and version that identifies the library sending request for debugging purposes.
    The default tracking source is ``jstc`` and can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```

Reply:
We don't collect source data only for debugging, we might at some point show a report with sources. How about this?
```suggestion
    Returns tracking source name and version that identifies the library sending tracking requests.
    This data will be passed in ``ts_n`` and ``ts_v`` tracking parameters.
    The default tracking source is ``jstc`` and can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```

Reply:
OK, just drop the specifics about `ts_n`` and ``ts_v`` tracking parameters. We don't mention how data is sent in other parts of documentation.

Reply:
```suggestion
    Returns tracking source name and version that identifies the library sending tracking requests.
    The default tracking source is ``jstc`` and can be overwritten using :ref:`setTrackingSource<jtc-api-setTrackingSource>` function.
```